docs/blog/posts/03-deploy-llms-on-a-budget.md

```diff
@@ -0,0 +1,319 @@
+
+## 🧠 What is Phixtral?
+
+[mlabonne/phixtral-4x2_8](https://huggingface.co/mlabonne/phixtral-4x2_8) is the first Mixure of Experts (MoE) made with 4 [microsoft/phi-2](https://huggingface.co/microsoft/phi-2) models, inspired by the [mistralai/Mixtral-8x7B-v0.1](https://huggingface.co/mistralai/Mixtral-8x7B-v0.1) architecture. The general idea behind mixture-of-experts is to combine the capabilities of multiple models to achieve better performance than each individual model. In this case, we combine the capabilities of 4 [microsoft/phi-2](https://huggingface.co/microsoft/phi-2) models to achieve better performance than each of the individual 2.7B parameter models it's composed of. 
+
+??? note "Breakdown of the [mlabonne/phixtral-4x2_8](https://huggingface.co/mlabonne/phixtral-4x2_8) model"
+
+    Here's the breakdown of the 4 models that make up the [mlabonne/phixtral-4x2_8](https://huggingface.co/mlabonne/phixtral-4x2_8) model:
+
+    ```yaml
+    base_model: cognitivecomputations/dolphin-2_6-phi-2
+    gate_mode: cheap_embed
+    experts:
+      - source_model: cognitivecomputations/dolphin-2_6-phi-2
+        positive_prompts: [""]
+      - source_model: lxuechen/phi-2-dpo
+        positive_prompts: [""]
+      - source_model: Yhyu13/phi-2-sft-dpo-gpt4_en-ep1
+        positive_prompts: [""]
+      - source_model: mrm8488/phi-2-coder
+        positive_prompts: [""]
+    ```
+
+    You can go to the original model card [here](https://huggingface.co/mlabonne/phixtral-4x2_8) for more details on how the model was merged using [mergekit](https://github.com/cg123/mergekit).
+
+Now, let's take a look at the performance of the [mlabonne/phixtral-4x2_8](https://huggingface.co/mlabonne/phixtral-4x2_8) model on the [Nous Suite](https://github.com/mlabonne/llm-autoeval?tab=readme-ov-file#evaluation-parameters) compared to other models in the 2.7B parameter range. 
+
+| Model | AGIEval	| GPT4All	| TruthfulQA	| Bigbench	| Average | 
+| --- | --- | --- | --- | --- | --- |
+| **[mlabonne/phixtral-4x2_8](https://huggingface.co/mlabonne/phixtral-4x2_8)** | **33.91** | 70.44 | 48.78 | **37.82** | **47.78** |
+| [dolphin-2_6-phi-2](https://huggingface.co/cognitivecomputations/dolphin-2_6-phi-2) | 33.12 | 69.85 | 47.39 | 37.2 | 46.89 |
+| [phi-2-dpo](https://huggingface.co/lxuechen/phi-2-dpo) | 30.39 | **71.68** | **50.75** | 34.9 | 46.93 |
+| [phi-2](https://huggingface.co/microsoft/phi-2) | 27.98 | 70.8 | 44.43 | 35.21 | 44.61 |
+
+
+## 💸 Serving Phixtral on a budget with SkyPilot and NOS
+
+Let's now see how we can deploy the [mlabonne/phixtral-4x2_8](https://huggingface.co/mlabonne/phixtral-4x2_8) model on a single NVIDIA L4 GPU for under $160 / month. We'll be using [SkyPilot](https://skypilot.readthedocs.io/) to deploy and manage our [NOS](https://github.com/autonomi-ai/nos) service on spot (pre-emptible) instances, making them especially cost-efficient.
```

Comment:
Might be good to mention cross cloud functionality available via skypilot, particularly if skypilot can automatically select an instance for us.

Reply:
Done.